### PR TITLE
Update draft-tjjk-cared.md

### PR DESCRIPTION
**Pull request by jess-krynitsky**
added callout to TLS 1.3 and PSK, with certificates recommended

### CODE REVIEW

**Comment by tojens-ietf** [comment]
@jdamick what do you think? I like this because it clarifies why we make the recommendation and encodes a key advantage that could be lost when using PSK without forbidding it.

**Comment by jess-krynitsky** [comment]
[like]  Jessica Krynitsky reacted to your message:
________________________________

File: draft-tjjk-cared.md [hunk 1]
```diff
@@ -270,8 +270,11 @@ organization via PKI heiracrchy
 for authentication
 
 Encrypted DNS clients and servers that support offering or requesting
-client authentication MUST support at least the use of mTLS in
-addition to whatever other mechanism they wish to support.
+client authentication MUST support at least the use of mTLS with TLS 1.3 in
+addition to whatever other mechanism they wish to support. Client 
+authentication using PKI certificates is recommended, but pre-shared
+keys may also be used, provided (EC)DHE key exchange is used to maintain
```
Comment (jdamick) [inline draft-tjjk-cared.md]: capitalize "MAY" i think?

Comment (ableyjoe) [reply draft-tjjk-cared.md]: also capitalise RECOMMENDED perhaps

Comment (tojens-ietf) [reply draft-tjjk-cared.md]: Updated, both are good points.

**Pull request merged**